packages/bot-web-ui/jest.config.js

```diff
@@ -0,0 +1,22 @@
+const baseConfigForPackages = require('../../jest.config.base');
+
+module.exports = {
+    ...baseConfigForPackages,
+    clearMocks: true,
+    moduleNameMapper: {
+        '\\.s(c|a)ss$': '<rootDir>/../../__mocks__/styleMock.js',
+        '^.+\\.svg$': '<rootDir>/../../__mocks__/styleMock.js',
+        '^App/(.*)$': '<rootDir>/src/app/$1',
+        '^Components/(.*)$': '<rootDir>/src/components/$1',
+        '^Constants/(.*)$': '<rootDir>/src/constants/$1',
+        '^Stores/(.*)$': '<rootDir>/src/stores/$1',
+        '^Utils/(.*)$': '<rootDir>/src/utils/$1',
+    },
+    coveragePathIgnorePatterns: [
```

Comment:
instead of this can we use `"collectCoverageFrom": ["./src/**"],`

Reply:
@ali-hosseini-deriv do you mean instead of "coveragePathIgnorePatterns" and "moduleNameMapper"?
I see in same places we use : `collectCoverageFrom: ['**/*.{js,jsx,ts,tsx}', '!**/node_modules/**']`
in same places as i wrote... Let me check how it will work with this way: `collectCoverageFrom: ['**/*.{js,jsx,ts,tsx}', '!**/node_modules/**']`

Reply:
not instead of `moduleNameMapper`
just remove `coveragePathIgnorePattern` and use `collectCoverageFrom`.

Reply:
so maybe better instead of "`collectCoverageFrom": ["./src/**"], ` add similar to documentation `collectCoverageFrom: ['**/*.{js,jsx,ts,tsx}', '!**/node_modules/**']`?
<img width="1098" alt="Screenshot 2023-03-23 at 13 27 04" src="https://user-images.githubusercontent.com/103181650/227175446-f272c0d6-e251-45ad-9779-281e930a2470.png">

Reply:
yeah sure, i just said that as an example. please go ahead with the approach that you mentioned

Reply:
thank you🙏